### PR TITLE
erights cookie

### PR DESCRIPTION
**Pull request by commuterjoy**
There's a few places where we can read this in the cookie.

### CODE REVIEW

File: src/models/User.js [hunk 1]
```diff
@@ -5,43 +6,46 @@
...
+};
 
 // Returns the eRights user id
 User.prototype.id = function () {
-    var i = this.cookie.match(/_EID=(\d+)_PID/);
-    return (i) ? i[1] : undefined; 
-}
+		var eid = this.cookie.match(/_EID=(\d+)_PID/);
+		var remember = this.cookie.match(/FT_Remember=(\d+):/);
+		var user = this.cookie.match(/USERID=(\d+):/);
```
Comment (commuterjoy) [inline src/models/User.js]: These three lines will help to find the erights id in the cookie :/

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "karma-bro": "^0.10.0",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^0.1.5",
-    "karma-mocha": "^0.1.9"
+    "karma-mocha": "^0.1.9",
+    "origami-build-tools": "*"
```
Comment (matthew-andrews) [inline package.json]: living on the edge

Comment (commuterjoy) [inline package.json]: stole it from dobi :)

**Pull request merged**